src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -560,8 +561,8 @@ export default class AircraftModel {
 
     initFms(data) {
         const airport = AirportController.airport_get();
-        const initialRunway = airport.getActiveRunwayForCategory(this.category);
-        this.fms = new Fms(data, initialRunway, this.model, this._navigationLibrary);
+        // const initialRunway = airport.getActiveRunwayForCategory(this.category);
+        this.fms = new Fms(data, this.model, this._navigationLibrary);
```

Comment:
To answer the `TODO` question about the `navigationLibrary`; yes, the `AircraftModel` does need a reference to it 😉

Reply:
this has been changed to `navigationLibrary` as a local variable passed down the initialization chain, after which it dies! :)